test/inductor/test_codecache.py

```diff
@@ -806,14 +788,28 @@ def f(x, val):
     @config.patch({"fx_graph_remote_cache": False})
     @config.patch({"freezing": True})
     @parametrize("device", (GPU_TYPE, "cpu"))
-    def test_freezing(self, device):
+    @parametrize("inlinable", (True, False))
+    def test_freezing(self, device, inlinable):
         if device == GPU_TYPE and not HAS_GPU:
             raise unittest.SkipTest(f"requires {GPU_TYPE}")
 
+        # For machines with mkldnn_fp16 support, weight_pack in mkldnn_fusion.py causes
```

Comment:
Somewhat unrelated. The mkldnn special handling below was added after I built this test. I just thought it was making the test overly complicated. If we're bypassing the cache when we find mkldnn tensors, there's not much to test so let's just skip.